docs/architecture/microservices/secure-net-microservices-web-applications/azure-key-vault-protects-secrets.md

```diff
@@ -21,7 +21,7 @@ The **Microsoft.Extensions.Configuration.AzureKeyVault** package allows an ASP.N
    Set-AzKeyVaultAccessPolicy -VaultName "<VaultName>" -ServicePrincipalName $sp.ServicePrincipalNames[0] -PermissionsToSecrets all -ResourceGroupName "<KeyVault Resource Group>"
    ```
 
-3. Include the key vault as a configuration source in your application by calling the <xref:Microsoft.Extensions.Configuration.AzureKeyVaultConfigurationExtensions.AddAzureKeyVault%2A?displayProperty=nameWithType> extension method when you create an <xref:Microsoft.Extensions.Configuration.IConfigurationRoot> instance. Note that calling `AddAzureKeyVault` requires the application ID that was registered and given access to the key vault in the previous steps.
+3. Include the key vault as a configuration source in your application by calling the AzureKeyVaultConfigurationExtensions.AddAzureKeyVault extension method when you create an <xref:Microsoft.Extensions.Configuration.IConfigurationRoot> instance. Note that calling `AddAzureKeyVault` requires the application ID that was registered and given access to the key vault in the previous steps.
```

Comment:
The application ID is not required when using the defaultazurecredential. Also, there is no cert specific overload anymore and we don't recommend referencing Microsoft.IdentityModel.Clients.ActiveDirectory package.

Reply:
Updated, please help review